src/sinks/file/mod.rs

```diff
@@ -33,22 +35,33 @@ use crate::{
     template::Template,
 };
 mod bytes_path;
-use std::convert::TryFrom;
 
 use bytes_path::BytesPath;
 
 /// Configuration for the `file` sink.
+#[serde_as]
 #[configurable_component(sink("file"))]
 #[derive(Clone, Debug)]
 #[serde(deny_unknown_fields)]
 pub struct FileSinkConfig {
     /// File name to write events to.
```

Comment:
the setting is called path but we call it a file name here. Seems like we should update this to indicate it is indeed a path.

Reply:
Updated in [d46dd77](https://github.com/vectordotdev/vector/pull/16357/commits/d46dd77b31495a5e32921e599eb5f86a8b48c25f)